sass/hugo-uswds/src/styles/components/_syntax-highlight.scss

```
@@ -27,13 +25,12 @@ div.highlight {
     @include u-padding-x(1);
     @include u-margin(0);
     display: block;
-    background-color: inherit;
     box-shadow: none;
-  
+    background-color: whitesmoke!important;
```

This appears to be a random color. This should use the USWDS color schemes.
```suggestion
    @include u-bg('base-lightest');
```